packages/data/addon/adapters/dimensions/elide.ts

```diff
@@ -8,22 +8,93 @@ import { getOwner } from '@ember/application';
 import EmberObject from '@ember/object';
 import NaviDimensionAdapter, { DimensionFilter } from './interface';
 import { ServiceOptions } from 'navi-data/services/navi-dimension';
-import { RequestV2 } from '../facts/interface';
-import ElideFactsAdapter from '../facts/elide';
+import { AsyncQueryResponse, FilterOperator, QueryStatus, RequestV2 } from '../facts/interface';
+import ElideFactsAdapter, { getElideField } from '../facts/elide';
 import { DimensionColumn } from 'navi-data/models/metadata/dimension';
 import ElideDimensionMetadataModel from 'navi-data/models/metadata/elide/dimension';
+import { assert } from '@ember/debug';
+
+type EnumFilter = (values: string[], filterValues: (string | number)[]) => (string | number)[];
+
+const enumOperators: Partial<Record<FilterOperator, EnumFilter>> = {
+  in: (values, filterValues) => values.filter(value => filterValues.includes(value)),
+  eq: (values, filterValues) => values.filter(value => filterValues[0] === value),
+  contains: (values, filterValues) =>
+    values.filter(value => ('' + value).toLowerCase().includes(('' + filterValues[0]).toLowerCase()))
```

Comment:
```suggestion
    values.filter(value => `${value}`.toLowerCase().includes(`${filterValues[0]}`.toLowerCase()))
```

Reply:
Last time I checked I thought the original was faster. However, according to jsbench.me, template string is faster. 👍